source/common/router/config_impl.h

```diff
@@ -83,6 +109,7 @@ class VirtualHostImpl : public VirtualHost {
   const ConfigImpl& globalRouteConfig() const { return global_route_config_; }
 
   // Router::VirtualHost
+  const CorsPolicy& corsPolicy() const override { return cors_policy_; }
```

Comment:
I was never sure if the ordering is important. BUILD files where alphabetical but the code wasn't.

Reply:
I don't think we bother for function names.  includes and buildfiles yes though :-)